src/config/language.js

```diff
@@ -0,0 +1 @@
+module.exports = { defaultLang: "yoruba", };
```

Comment:
based on the comment below, this file will also no longer be necessary. Because we will have ```global.defaultLang```